test/utils/bootstrapUtilsSpec.js

```diff
@@ -191,7 +192,8 @@ describe('bootstrapUtils', ()=> {
     });
 
     it('should work with createClass', ()=> {
-      shouldWarn('expected one of ["smallish","micro","planet"]');
+      // this triggers an error: the warning list is empty
+      // shouldWarn('expected one of ["smallish","micro","planet"]');
```

Comment:
Yep, it is most likely unrelated to Babel, probably due to that I ran an `npm-check-updates` on the branch...